samples/bluetooth/hci_rpmsg/nrf5340_cpunet_iso_receive.conf

```diff
@@ -22,22 +22,24 @@ CONFIG_BT_OBSERVER=y
 CONFIG_BT_CENTRAL=n
 CONFIG_BT_EXT_ADV=y
 # CONFIG_BT_PER_ADV=y
-# CONFIG_BT_ISO_BROADCASTER=y
 CONFIG_BT_PER_ADV_SYNC=y
+# CONFIG_BT_ISO_BROADCASTER=y
```

Comment:
Remove? 
```suggestion
```

Reply:
ditto, they are here to be able to support both broadcaster and receiver together.